content/game-controllers/vkb/create-definition/index.md

```
@@ -0,0 +1,23 @@
+- Base, including everything listed in the description
+- Grips, starting from left to right for throttle grips
+- SEMs, including SEM-Vs
+- THQs, including THQ-Vs. If one of the THQs has a WW2 throttle grip, choose that one last.
+- STEM
+- FSM.GA
+
+It is best to verify button numbers in a button tester application.
+
+Once the program has completed, a file ending in `.joystick.json` will be available in the `output` folder. Copy this file to the `Joysticks` folder in your MobiFlight installation directory, e.g. `%LOCALAPPDATA%\MobiFlight\MobiFlight Connector\Joysticks`.
+
+If you are running a heavily customized VKBDevCfg configuration, it may be necessary to make manual adjustments to the file to ensure button labels are accurate.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)